Library/Homebrew/utils/ruby.sh

```diff
@@ -1,3 +1,8 @@
+test-ruby () {
+  "$1" --enable-frozen-string-literal --disable=gems,did_you_mean,rubyopt -rrubygems -e \
+    "puts Gem::Version.new(RUBY_VERSION.to_s.dup).to_s.split('.').first(2) == Gem::Version.new('$required_ruby_version').to_s.split('.').first(2)"
```

Comment:
What about breaking this line down as well so that the second `Gem::Version.new` appears under the first one?